src/core/schema/src/main/scala/ImageSchema.scala

```diff
@@ -3,21 +3,21 @@
 
 package com.microsoft.ml.spark.schema
 
-import com.microsoft.ml.spark._
-import org.apache.spark.rdd.RDD
-import org.apache.spark.sql.{DataFrame, Row}
 import org.apache.spark.sql.types._
-import scala.reflect.ClassTag
+import org.apache.spark.sql.{DataFrame, Row}
 
 object ImageSchema {
 
   /** Schema for the image column: Row(String, Int, Int, Int, Array[Byte]) */
-  val columnSchema = StructType(
-    StructField("path",   StringType,  true) ::
-    StructField("height", IntegerType, true) ::
-    StructField("width",  IntegerType, true) ::
-    StructField("type", IntegerType, true) ::                 //OpenCV type: CV_8U in most cases
-    StructField("bytes",  BinaryType, true) :: Nil)   //OpenCV bytes: row-wise BGR in most cases
+  val internalSchema = StructType(
+    StructField("path",     StringType,  true) ::
+      StructField("height", IntegerType, true) ::
+      StructField("width",  IntegerType, true) ::
+      StructField("type",   IntegerType, true) ::                 //OpenCV type: CV_8U in most cases
```

Comment:
This is a subtle change to ``ImageSchema`` correct? Given it's getting merged with the spark images work, why not make it there?